.github/workflows/ci.yml

```
@@ -1,26 +1,21 @@
 name: CI
 
 on:
-  push:
-    branches-ignore:
-      - "github-pages/*"
-      - "gh-pages/*"
-      - "main"
-      - "master"
+  pull_request:
+      types: [opened, synchronize, reopened]
   schedule:
     # “At 00:00 on every 7th day-of-month from 1 through 31.” (https://crontab.guru)
```

source of copy-paste issue :cry: 
```suggestion
  # At 00:00 on every 7th day-of-month from 1 through 31. (https://crontab.guru)
```